slack-infra/resources/ingress.yaml

```diff
@@ -25,3 +25,7 @@ spec:
             backend:
               serviceName: slack-welcomer
               servicePort: 80
+          - path: /infra/post-message/*
+            backend:
+              serviceName: slack-post-message
+              servicePort: 80
```

Comment:
@rajula96reddy just a nit, could  you add the new line at the end?

Reply:
Yeah I will do it right away